### ISSUE TITLE
Clearing blurs on first image of the sequence exits the editor

### ISSUE THREAD

**Issue by kaitu** [mapillary/mapillary_issues#2353]
### Basic information
* Platform: `Web`
* System/Browser: `Firefox 49.0.2`

### Steps to reproduce behavior

 1. Find a sequence with blurs on the first image
 2. Press the hamburger button and call the blur editor
 3. Stay on the very same photo on which you called the blur editor
 4. Press the "clear all" button

### Expected behavior
the blur editor stays open and moves automatically to next picture in the sequence.

### Actual behavior
automatic exit from blur editor. If I press the hamburger button, the "Photo details" window says "Blur approval is pending on this photo" (which is correct). I then need to move to next photo and reopen the blur editor to continue the review.

**Comment by gyllen** [comment]
duplicate of #2352 closing

**Comment by HolgerJeromin** [comment]
Very related. But really the same?

**Comment by gyllen** [comment]
@HolgerJeromin Yeah I think fixing #2352 will fix this issue to, let us know and reopen if it doesn't
